usaspending_api/awards/management/commands/loadcontracts.py

```diff
@@ -18,6 +18,12 @@ def add_arguments(self, parser):
         parser.add_argument('file', nargs=1, help='the file to load')
 
     def handle(self, *args, **options):
+        # Create a new submission attributes object for this timestamp
+        subattr = SubmissionAttributes()
```

Comment:
@afrasier ok cool--I added [DS-535](https://federal-spending-transparency.atlassian.net/browse/DS-535) so we don't forget to test this out.